src/simple_tracker_interfaces/msg/TrackTrajectory.msg

```diff
@@ -0,0 +1,2 @@
+string id
```

Comment:
Is this a UUID? Comments needed.

Reply:
No, its a simple integer counter of the track or bounding box. It is the same type as the id that is defined in https://github.com/ros-perception/vision_msgs/blob/ros2/msg/Detection2D.msg. I do use it to also transfer the tracking state over as we are using a standard message type that does not include that sort of attribute. I'll see about adding comments to the interface objects as well.

Reply:
Why is a string used instead of an integer? Consider using enumerations instead of bare data where appropriate.

Reply:
I don’t have control over the community message types, all I have done is try and use the same Id type in ours so that they match. Also, not sure how to do enumerations in message types.